policyengine_us/variables/household/demographic/tax_unit/senior_head.py

```diff
@@ -0,0 +1,14 @@
+from policyengine_us.model_api import *
+
+
+class senior_head(Variable):
+    value_type = bool
+    entity = TaxUnit
+    definition_period = YEAR
+    label = "Tax unit head is a senior"
+
+    def formula(tax_unit, period, parameters):
+        person = tax_unit.members
+        senior = person("is_senior", period)
```

Comment:
This is a somewhat arbitrary categorization for the purposes of poverty age breakdowns.

If the law defines age 65 for eligibility, please add a parameter specifying age 65.